docs/src/design-system/building-blocks/button/ButtonStatesExample.tsx

```diff
@@ -0,0 +1,253 @@
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  tableCellClasses,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Theme,
+  ThemeProvider,
+  useTheme,
+} from '@mui/material';
+import { Button, Typography } from '@ui-kit-2022/components';
+import React from 'react';
+
+export interface Props {
+  variantType?: 'PRIMARY' | 'SECONDARY' | 'TERTIARY';
+  theme: Theme;
+}
+
+const ButtonStatesExample: React.FC<Props> = ({ variantType, theme = useTheme() }) => {
+  return (
+    <TableContainer>
+      <Table
+        sx={{
+          [`& .${tableCellClasses.root}`]: {
+            borderBottom: 'none',
+            p: '0.625rem 0 0.625rem 0.5rem',
+          },
+          width: '78%',
+        }}
+      >
+        <TableHead>
+          <TableRow>
+            <TableCell align="left">
+              <Typography variant="subheader1"></Typography>
+            </TableCell>
+            <TableCell align="left">
+              <Typography variant="subheader3">Default</Typography>
+            </TableCell>
+            <TableCell align="left">
+              <Typography variant="subheader3">Hover</Typography>
+            </TableCell>
+            <TableCell align="left">
+              <Typography variant="subheader3">Active/Pressed</Typography>
+            </TableCell>
+            <TableCell
+              align="left"
+              sx={{
+                paddingRight: '0',
+                paddingTop: '0',
+                //color: palette.common.white,
+              }}
+            >
+              <Typography variant="subheader3">Focused(ADA)</Typography>
+            </TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          <TableRow>
+            <TableCell
+              sx={{
+                width: '12%',
+              }}
+            >
+              <Typography variant="subheader3">Default</Typography>
+            </TableCell>
+            <ThemeProvider theme={theme}>
+              <TableCell
+                sx={{
+                  backgroundColor: {
```

Comment:
It'll be better if you assign `const backgroundColor = ...` once at the top of the component and then you can re-use the variable.